test/test_end_point.py

```diff
@@ -1,12 +1,18 @@
 import unittest
+import sys
+from os.path import dirname, abspath
+
+sys.path.append(dirname(dirname(abspath(__file__))))
+
 from src.end_point import EndPoint
 
 
 class TestEndPoint(unittest.TestCase):
     def setUp(self):
         json_dict = {
             "name": "Get employee details",
-            "_postman_id": "385f7848-62db-4435-b7cf-820c3e7e5097",
+            "_postmian_id": "385f7848-62db-4435-b7cf-820c3e7e5097",
```

Comment:
is this supposed to be "_postman_id"?

Reply:
Actually yeah, but in latest Python versions the variable isn't showing up in collections. So I changed _postman_id to id and it is running fine.

Reply:
Oh right the name is wrong! Fixed it! Thx

Reply:
Just asking because it is currently "_postmain_id" not "_postman_id" not sure if that spelling matters though

Reply:
It was a spelling mistake I made. Will fix it in the next commits.